src/deepsparse/transformers/helpers.py

```diff
@@ -111,12 +112,20 @@ def get_onnx_path_and_configs(
         config_path = _get_file_parent(
             zoo_model.deployment.default.get_file(_MODEL_DIR_CONFIG_NAME).path
         )
-        tokenizer_path = _get_file_parent(
-            zoo_model.deployment.default.get_file(_MODEL_DIR_TOKENIZER_NAME).path
-        )
         tokenizer_config_path = zoo_model.deployment.default.get_file(
             _MODEL_DIR_TOKENIZER_CONFIG_NAME
         )
+        tokenizer_path = _get_file_parent(
+            zoo_model.deployment.default.get_file(_MODEL_DIR_TOKENIZER_NAME).path
+        )
+        if tokenizer_path is None:
```

Comment:
what do we do if tokenizer config is also missing?

Reply:
tokenizer config is not mandatory?

Reply:
tokenizer config should always be present. are you suggesting to introduce an explicit error-raising logic if this is the case, Ben?